python/taichi/lang/matrix.py

```diff
@@ -1880,6 +1883,9 @@ def filled_with_scalar(self, value):
     def field(self, **kwargs):
         return Vector.field(self.n, dtype=self.dtype, **kwargs)
 
+    def shape(self):
+        return (self.n, )
+
```

Comment:
Add a `shape` method for MatrixType and VectorType to replace verbose expressions (dtype.n, ) and (dtype.m, dtype.n). Does it look good here? @ailzhang